src/structures/Message.js

```diff
@@ -510,7 +510,11 @@ class Message extends Base {
             
             const canRevoke = window.Store.MsgActionChecks.canSenderRevokeMsg(msg) || window.Store.MsgActionChecks.canAdminRevokeMsg(msg);
             if (everyone && canRevoke) {
-                return window.Store.Cmd.sendRevokeMsgs(chat, [msg], { clearMedia: true, type: msg.id.fromMe ? 'Sender' : 'Admin' });
+                if (window.WWebJS.compareWwebVersions(window.Debug.VERSION, '>', '2.3000.0')) {
```

Comment:
```suggestion
                if (window.WWebJS.compareWwebVersions(window.Debug.VERSION, '>=', '2.3000.0')) {
```

New approach starts from v2.3000.0 inclusive